src/utils/sources-tree.js

```diff
@@ -73,14 +73,31 @@ function createParentMap(tree: any): WeakMap<any, any> {
  * @memberof utils/sources-tree
  * @static
  */
+function getFilenameFromPath(pathname) {
+  let filename = "";
+  if (pathname) {
```

Comment:
You could use `pathname?: string` instead to indicate that you are going to handle the null / undefined case in the code.